### PR TITLE
[AMS-2020] Open registration and update the welcome page

### PR DESCRIPTION
**Pull request by dpnl87**
Update the welcome page to make it shine and open up the registration.

### CODE REVIEW

File: content/events/2020-amsterdam/registration.md [hunk 1]
```diff
@@ -4,8 +4,16 @@ Type = "event"
...
 +++
 
-<div style="width:100%; text-align:left;">
-
-Embed registration iframe/link/etc.
-</div></div>
+<div class="container-fluid">
+    <div class="row justify-content-start">
+        <div class="col-md-8">
+            If you or your company is interested in paying us directly via an invoice for a bulk ticket order, please <a href="mailto:amsterdam@devopsdays.org?subject=devopsdays%20Amsterdam%202020%20-%20Pay%20Direct%20with%20Invoice">contact us by sending us an email @ eindhoven@devopsdays.org</a>
```
Comment (bridgetkromhout) [inline content/events/2020-amsterdam/registration.md]: eindhoven@devopsdays.org?

File: content/events/2020-amsterdam/welcome.md [hunk 2]
```diff
@@ -5,83 +5,62 @@ aliases = ["/events/2020-amsterdam/"]
...
-<div class = "row">
-  <div class = "col-md-2">
-    <strong>Contact</strong>
-  </div>
-  <div class = "col-md-8">
-    {{< event_link page="contact" text="Get in touch with the organizers" >}}
+  <div class="col-md-7">
+    <div class="col-sm-12">
+      <h2>What is devopsdays Amsterdam</h2>
+      From fledgling startups to established industry, the The Netherlands is home to a large, vibrant technology
```
Comment (bridgetkromhout) [inline content/events/2020-amsterdam/welcome.md]: the The?

File: content/events/2020-amsterdam/welcome.md [hunk 3]
```diff
@@ -5,83 +5,62 @@ aliases = ["/events/2020-amsterdam/"]
...
+      community. Amsterdam, in particular, has been a flurry of activity in the past several years, drawing attention from
+      around the globe.
+      <br><br>
+      devopsdays Amsterdam brings development, operations, QA, InfoSec, management, and leadership together to discuss the
+      culture and tools to make better organizations and products. The 2020 event will be the eight time we have held
```
Comment (bridgetkromhout) [inline content/events/2020-amsterdam/welcome.md]: eight -> eighth (yes, English is terri-bad)

File: content/events/2020-amsterdam/registration.md (hunk 1 above)
Comment (dpnl87) [reply content/events/2020-amsterdam/registration.md]: Fixed the link, forgot the text. Thnx!

**Comment by bridgetkromhout** [comment]
LGTM now - unsure if @yvovandoorn has additional changes he'd like to see. If he's happy with it then this can be merged.

**Comment by yvovandoorn** [comment]
LGTM :shipit:

**Pull request merged**